components/camel-aws/src/main/java/org/apache/camel/component/aws/sqs/SqsProducer.java

```diff
@@ -56,33 +57,18 @@ public void process(Exchange exchange) throws Exception {
         SendMessageRequest request = new SendMessageRequest(getQueueUrl(), body);
         request.setMessageAttributes(translateAttributes(exchange.getIn().getHeaders(), exchange));
         addDelay(request, exchange);
-        configureFifoAttributes(request, exchange);
 
         LOG.trace("Sending request [{}] from exchange [{}]...", request, exchange);
-
+        
         SendMessageResult result = getClient().sendMessage(request);
-
+        
         LOG.trace("Received result [{}]", result);
-
+        
         Message message = getMessageForResponse(exchange);
         message.setHeader(SqsConstants.MESSAGE_ID, result.getMessageId());
         message.setHeader(SqsConstants.MD5_OF_BODY, result.getMD5OfMessageBody());
     }
 
-    private void configureFifoAttributes(SendMessageRequest request, Exchange exchange) {
-        if (getEndpoint().getConfiguration().isFifoQueue()) {
-            // use strategies
-            MessageGroupIdStrategy messageGroupIdStrategy = getEndpoint().getConfiguration().getMessageGroupIdStrategy();
-            String messageGroupId = messageGroupIdStrategy.getMessageGroupId(exchange);
-            request.setMessageGroupId(messageGroupId);
-
-            MessageDeduplicationIdStrategy messageDeduplicationIdStrategy = getEndpoint().getConfiguration().getMessageDeduplicationIdStrategy();
-            String messageDeduplicationId = messageDeduplicationIdStrategy.getMessageDeduplicationId(exchange);
-            request.setMessageDeduplicationId(messageDeduplicationId);
-
-        }
-    }
-
     private void addDelay(SendMessageRequest request, Exchange exchange) {
```

Comment:
Why are you removing this block?

Reply:
Sorry I had messed up. Please check this again.